specs/interop/supervisor.md

```diff
@@ -0,0 +1,70 @@
+# Supervisor
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [RPC API](#rpc-api)
+  - [`supervisor_checkMessage`](#supervisor_checkmessage)
+    - [Parameters](#parameters)
+    - [Returns](#returns)
+  - [`supervisor_superRootAtTimestamp`](#supervisor_superrootattimestamp)
+    - [Parameters](#parameters-1)
+    - [Returns](#returns-1)
+    - [Description](#description)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+The supervisor is an implementation detail of OP Stack native interop and is not the only
+architecture that can be used to implement it. The supervisor is responsible for indexing
+data from all of the chains in a cluster so that the [safety level](./verifier.md#safety)
+of executing messages can quickly be determined.
+
+## RPC API
+
+### `supervisor_checkMessage`
+
+Checks the safety level of a specific message based on its identifier and message hash.
+This RPC is useful for the block builder to determine if a message is should be included
+in a block.
```

Comment:
It looks to me like there are a few more RPC methods defined in the `op-supervisor`. Noting this so they are not forgotten if needed.

The `op-supervisor` defines the set of query methods [here](https://github.com/ethereum-optimism/optimism/blob/develop/op-supervisor/supervisor/frontend/frontend.go#L18-L28).

These methods are then exposed as RPC methods by the server [here](https://github.com/ethereum-optimism/optimism/blob/develop/op-supervisor/supervisor/service.go#L165-L169) under the `supervisor_*` prefix.